pkg/fileutils/fileutils.go

```diff
@@ -94,7 +94,18 @@ func Link(src string, linkName string) error {
 
 // SymbolicLink creates target as a symbolic link to src.
 func SymbolicLink(src string, target string) error {
-	// TODO Add verifications.
+	if !PathExist(src) {
+		return fmt.Errorf("Symlink %s to %s: error, src no such file or directory", target, src)
+	}
+	if PathExist(target) {
+		if IsDir(target) {
+			return fmt.Errorf("Symlink %s to %s: error, link name already exists and is a directory", target, src)
+		}
+		if err := DeleteFile(target); err != nil {
+			return err
```

Comment:
why not to add more details in the error message rather than directly return the error? 
for example:
```
return fmt.Errorf("failed to symlink %s to %s when deleting target file: %v", target, src, err)
```

Reply:
Done.